### DISCUSSION TITLE
How dow you suggest changes for default config templates?

### DISCUSSION THREAD

**Discussion by wayner9** [arendst/Tasmota#19946] [Q&A] [answered]
For the default template for the  Sonoff THR320D, GPIO 15 is set for Led_Link_i.  I think it should be Led_Link.  This way the Wifi LED stays on when you have a valid Wifi connection, which is presumably the desired action.  How do you suggest this change so that the default template on Blakadder, or other places, is corrected for the benefit of others in the future?

**Comment by barbudor** [comment]
You need to create a PR on https://github.com/blakadder/templates
Specifically the file for Sonoff THR320D is there: https://github.com/blakadder/templates/blob/cb208ab402a0c22f27f021bd574e183261f64616/_templates/sonoff_THR320D#L4

**Reply by wayner9** [reply]
What's a PR?

**Reply by barbudor** [reply, marked as answer]
A Pull Request
It is a process in Github to suggest changes on a file
You need to:
- Fork the "templates" repository => that will make a copy in your own account
       go to:  https://github.com/blakadder/templates
       on top of page, click on "Fork"      
![image](https://github.com/arendst/Tasmota/assets/2996042/a9bccb6e-83fd-4491-aa9f-19d9c014ea82)

- In your copy, go to the same file. Should be https://github.com/wayner9/templates/blob/master/_templates/sonoff_THR320D
- Use the pencil button to make the change
![image](https://github.com/arendst/Tasmota/assets/2996042/ae74fc75-2610-46d0-b1b0-f808299cb3a6)
- Save using the "Commit changes..." button
![image](https://github.com/arendst/Tasmota/assets/2996042/dde9196b-52fb-4cf2-9204-64c6dd495361)
- Then you will be proposed to create a Pull Request

**Reply by wayner9** [reply]
Ok, thanks.  This stuff is confusing when doing it for the first time.

**Reply by sfromis** [reply]
Each template page already has an edit link, to the right, for Github page with the device. This route will automatically fork when you edit, and generally have simpler steps, directly available on each page you get to.

**Reply by wayner9** [reply]
Thanks, I had never done this before but now I have submitted the change via a Pull Request.  But when I click on Pull Request from the link at the top of this page I don't see it under the list of Pull Requests.

**Reply by sfromis** [reply]
Clicking "Pull Requests" is to see if someone else has been submitting a PR for you to deal with. It does not look like you actually submitted a PR to https://github.com/blakadder/templates/pulls and your commit https://github.com/blakadder/templates/compare/master...wayner9:templates:master still has a button for "Create pull request".

Anyway, while your change may reflect your preference for how you like the led to work, this may not be a presumption for users in general. It's the type of thing where there is no "correct" of led behavior. However, I have no strong opinion either way. :smile:

**Reply by wayner9** [reply]
Maybe there is no one correct behaviour but if there is a light called Wifi (or it has the Wifi symbol) then shouldn't it be lit when you have a Wifi connection live?

**Reply by sfromis** [reply]
If that makes sense to you.... To me it makes more sense to have it blink in case of trouble.
